src/App.js

```diff
@@ -34,14 +39,24 @@ class App extends Component {
         googleId: localStorageObject.googleId
       })
     }
+    
+    var userInfo = firebase.database().ref('users/' + localStorage.getItem('id')); // had to change to local storage b/c
```

Comment:
id doesnt change in state anymore i thought?

Reply:
yup, just removed those lines of code